pkg/mev/bundle_sender.go

```diff
@@ -239,25 +149,10 @@ func (s *Client) SendBackrunBundle(
 	uuid *string,
 	blockNumber uint64,
 	pendingTxHash common.Hash,
+	_ []string,
 	txs ...*types.Transaction,
 ) (SendBundleResponse, error) {
-	switch s.senderType {
-	case BundleSenderTypeFlashbot:
-		if len(txs) == 0 {
-			return SendBundleResponse{}, fmt.Errorf("no transactions to send")
-		}
-		resp, err := s.flashbotBackrunSendBundle(blockNumber, pendingTxHash, txs[0])
-		if err != nil {
-			return SendBundleResponse{}, err
-		}
-		return SendBundleResponse{
-			Result: SendBundleResult{
-				BundleHash: resp.BundleHash.String(),
-			},
-		}, nil
-	default:
-		return s.ethBackrunSendBundle(ctx, uuid, blockNumber, pendingTxHash, txs...)
-	}
+	return s.ethBackrunSendBundle(ctx, uuid, blockNumber, pendingTxHash, txs...)
```

Comment:
This client only sends bundle don't know anything about backrun, frontrun, or sandwich.